R/handle_metadata.R

```diff
@@ -0,0 +1,33 @@
+#' @title Handle metadata from bundled packages.
+#'
+#' @description Will return the metadata from the GAMBLR.data package when it is
+#' installed. Otherwise, will instruct user to call
+#' GAMBLR.results::get_gambl_metadata().
+#'
+#' @details INTERNAL FUNCTION for handling the metadata.
+#'
+#' @return data frame
+#'
+#' @param this_seq_type The seq type of the samples to return with metadata.
+#'
+#' @import GAMBLR.data dplyr
+#'
+#' @noRd
+#'
+#' @export
+handle_metadata = function(this_seq_type = "genome") {
+    if ("GAMBLR.data" %in% installed.packages()) {
+        return(
+            GAMBLR.data::gambl_metadata %>%
```

Comment:
Should we add a message here, that states that the metadata is coming from the GAMBLR.data package?

Reply:
Good suggestion, will do!